torch/nn/functional.py

```diff
@@ -2908,8 +2909,8 @@ def binary_cross_entropy_with_logits(
     See :class:`~torch.nn.BCEWithLogitsLoss` for details.
 
     Args:
-        input: Tensor of arbitrary shape
-        target: Tensor of the same shape as input
+        input: Tensor of arbitrary shape as unnormalized scores
```

Comment:
maybe a comment like "unnormalized scores (often referred to as logits)" would be useful here and in `cross_entropy`